build/terraform/providers/aws/variables.tf

```diff
@@ -68,6 +68,31 @@ variable "ebs_csi_driver" {
   default     = "github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/stable/?ref=master"
 }
 
+variable "efs_csi_driver" {
+  description = "GitHub path to the CSI plugin for data persistence"
+  default     = "github.com/kubernetes-sigs/aws-efs-csi-driver/deploy/kubernetes/overlays/stable/?ref=master"
+}
+
+variable "efs_pv_sifnode_name" {
+  description = "The name of the PV object"
+  default     = "efs-sifnode"
```

Comment:
Sorry to be anal, but thinking we change the name to `efs-sifnoded` ... thoughts?